.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not required, as well as the rest of the .vs folder

Reply:
When I was looking through the files/code I didn't see a `.vs` folder? Is it maybe in some hidden area or is it a format that my computer can't show (that sounds strange but it has happened to me before)?

Reply:
My bad, now I see it. Do you mean I can delete the **entire** `.vs` folder?

Reply:
Sorry about the long line of answers, but my computer is having some security issues, so I don't have the ability to delete the `.vs` folder for some reason.